sdk/search/Azure.Search.Documents/src/Indexes/Models/AzureMachineLearningSkill.cs

```diff
@@ -0,0 +1,319 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.Search.Documents.Indexes.Models
+{
+    /// <summary>
+    /// The Azure Machine Learning skill allows you to extend AI enrichment with a custom Azure Machine Learning (AML) model.
+    /// Once an AML model is trained and deployed, an AML skill integrates it into AI enrichment.
+    /// </summary>
+    public partial class AzureMachineLearningSkill : IUtf8JsonSerializable
+    {
+        /// <summary> (Required for key authentication) The key for the AML service. </summary>
```

Comment:
I don't think I've seen our ref docs start with parenthesized text, it feels like it'd be better to say what it is before we jump into how it's required, and it might be helpful to explain what it's used with?
```suggestion
        /// <summary>The key for the Azure Machine Learning service.  This is required for key-based authentication along with <ref-magic>ScoringUri</ref-magic>.</summary>
```